README.md

```diff
@@ -21,23 +23,45 @@
 
 # 🎯 How do I use the dbt package?
 ## Step 1: Prerequisites
-To use this dbt package, you must have the Fivetran **SAP** (sap.com) the respective tables to your destination:
-### SAP.com
-- bkpf
-- bseg
-- faglflexa
-- faglflext
-- mara
-- ska1
-- t001
+To use this dbt package, you must have the following:
+- At least one Fivetran of the following SAP connectors:
+   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
+   - [HVA SAP ECC](https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana)
+   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
+-  Within the connector, syncing the following respective tables into your destination:
+  - bkpf
+  - bseg
+  - faglflexa
+  - faglflext
+  - kna1
+  - lfa1
+  - mara
+  - pa0000
+  - pa0001
+  - pa0007
+  - pa0008
+  - pa0031
+  - ska1
+  - t001
+  - t503
+  - t880
+- A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, **Databricks** destination.
```

Comment:
The bullets don't seem to be consistent in the rendered preview. I think the second set of indented bullets need to be adjusted.
![image](https://github.com/fivetran/dbt_sap_source/assets/74217849/4e895f4d-1e2d-4574-ae9e-99891e9b0ca0)

```suggestion
- At least one Fivetran of the following SAP connectors:
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP ECC](https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
-  Within the connector, syncing the following respective tables into your destination:
   - bkpf
   - bseg
   - faglflexa
   - faglflext
   - kna1
   - lfa1
   - mara
   - pa0000
   - pa0001
   - pa0007
   - pa0008
   - pa0031
   - ska1
   - t001
   - t503
   - t880
- A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, **Databricks** destination.
```

Reply:
You apparently have to space when you're indenting a bullet point =P. This is fixed on both source and transform packages.